### DISCUSSION TITLE
Librilight_limited

### DISCUSSION THREAD

**Discussion by SNkhan12** [espnet/espnet#5733] [Q&A] [answered]
Hi,

This is my first time working with ESPnet, and I'm experiencing some difficulties with **librilight_limited**. When I run the run.sh script, I encounter the following error: "local/data.sh: line 61: ./utils/make_absolute.sh: No such file or directory." Can someone please clarify the purpose of this file?

**Comment by sw005320** [comment]
Did you follow the installation instructions and download Kaldi?
This script comes from https://github.com/kaldi-asr/kaldi/blob/master/egs/wsj/s5/utils/make_absolute.sh

**Reply by SNkhan12** [reply]
Yes I did. I have Kaldi installed already. 

But actually, I thought that ESPnet2 doesn't need Kaldi.

**Reply by sw005320** [reply, marked as answer]
Right. We don't need to "install" Kaldi, but some special recipes, including this, use the Kaldi scripts, and we need to "download" it.

Can you make sure that you correctly download Kaldi? 
Also, can you check the path?

**Reply by SNkhan12** [reply]
It works, I just had to add the path. Thank u sir.
